external/prices/daemon.prices.multicall.go

```diff
@@ -220,13 +180,10 @@ func setMissingYearnVaultPrices(chainID uint64) {
 				vaultValue = vaultValue.Div(vaultValue, bigNumber.NewInt(1e6))
```

Comment:
hmm if the `pricePerShare` here is used only for setting the `vaultValue`, perhaps we could get rid of the multiplication and division by 10^6 now